omnigibson/examples/objects/draw_bounding_box.py

```diff
@@ -67,22 +70,73 @@ def main(random_selection=False, headless=False, short_exec=False):
         env.step(np.array([]))
 
     # Grab observations from viewer camera and write them to disk
-    obs = cam.get_obs()
+    obs = cam.get_obs()[0]
 
     for bbox_modality in bbox_modalities:
         # Print out each of the modalities
         og.log.info(f"Observation modality {bbox_modality}:\n{obs[bbox_modality]}")
 
         # Also write the 2d loose bounding box to disk
         if "3d" not in bbox_modality:
-            colorized_img = lazy.omni.isaac.synthetic_utils.visualization.colorize_bboxes(bboxes_2d_data=obs[bbox_modality], bboxes_2d_rgb=obs["rgb"], num_channels=4)
+            colorized_img = colorize_bboxes(bboxes_2d_data=obs[bbox_modality], bboxes_2d_rgb=obs["rgb"], num_channels=4)
```

Comment:
is this more efficient than the omni native version? or is it just copied here for convenience? (or maybe something else?)

Reply:
I actually tried the replicator native version `colorize_bbox_2d` first but it keeps producing wrong output. I looked into this today a little further and I found their version of this function has a bug in there. 
Here's a part of their code:
```
y_min_x_min = [data["y_min"], data["x_min"]]
y_max_x_max = [data["y_max"], data["x_max"]]
for bbox_min, bbox_max, color in zip(y_min_x_min, y_max_x_max, colors):
            rgb_img_draw.rectangle(
                xy=[bbox_min[1], bbox_min[0], bbox_max[1], bbox_max[0]],
                outline=color,
                width=2,
            )
```
Essentially, their ordering of the rectangle xy is off. If we have two bounding boxes, this code would give something like:
bbox1: [y_min_0, y_min_1, y_max_0, y_max_1]
bbox2: [x_min_0, x_min_1, x_max_0, x_max_1]
which is just completely wrong. 
The `colorize_bboxes` and `random_colours` functions I have here are adapted from the synthetic data version of them and they work fine. I'm happy to move them over to a utils file. Let me know what you think!